eox_core/edxapp_wrapper/backends/enrollment_h_v1.py

```diff
@@ -67,3 +70,25 @@ def check_edxapp_enrollment_is_valid(*args, **kwargs):
         except CourseModeNotFoundError:
             errors.append('Mode not found')
     return errors
+
+
+def _create_or_update_enrollment(username, course_id, mode, is_active, try_update):
+    try:
+        enrollment = api._data_api().create_course_enrollment(username, course_id, mode, is_active)
+    except CourseEnrollmentExistsError as e:
+        if try_update:
+            enrollment = api._data_api().update_course_enrollment(username, course_id, mode, is_active)
+        else:
+            raise Exception(repr(e) + ", use force_registration to update the existing enrollment")
+    return enrollment
+
+
+def _force_create_enrollment(username, course_id, mode, is_active):
+    try:
+        course_key = CourseKey.from_string(course_id)
```

Comment:
Could you please log here under the INFO level that we used the force enrollment creation?